src/gui/properties/propertieswidget.ui

```diff
@@ -1017,14 +1017,14 @@
          <item>
           <widget class="QPushButton" name="selectAllButton">
            <property name="text">
-            <string>Select All</string>
+            <string>Check all items</string>
```

Comment:
@Chocobo1:  I'm just curious why do you object to the real action names ("Download all"/"Download none")? Why do you want to use a proxy action ("Check" or "Select")?